README.md

```diff
@@ -1,26 +1,20 @@
 # BuyTogether
-
 Welcome to our group social project!.
-In this project we have developed a web application which suppose to help people group up
-and organize a large scale food buy.
-Doing so, we "cut" the amount of hands the product go through which will benefit the customer
+In this project, we have developed a web application that suppose to help people group up,
+and organize a large-scale food buy.
+Doing so, we "cut" the amount of hands the product goes through, which will benefit the customer
```

Comment:
the grouping feature doesn't remove "the amount of hands ...", the direct access does even if it is a single user. The grouping helps get better pricing since the overall order is bigger. Write something along the lines of "...remove middlemen and reduce price ... "